resources/views/servidor/details.blade.php

```diff
@@ -34,6 +34,53 @@
             </div>
         </div>
     </div>
+    <div class="py-12">
```

Comment:
Esse padding 12 rem que está dando o espaçamento 
```suggestion
    <div class="py-4">
```